projects/sitemanage/src/test/java/com/percussion/share/test/PSMatchers.java

```diff
@@ -111,7 +111,7 @@ public boolean matchesSafely(String item)
                 {
                     stream = IOUtils.toInputStream(item, "UTF-8");
                 }
-                catch (IOException e)
```

Comment:
@rgarg24  Why the change here?  We should be capturing named exceptions not general exceptions if possible.

Reply:
@natechadwick yes Nate you are right. But in this case, IOException is never thrown due to which we were getting compile time error. That's why I made it general exception.

Reply:
@rgarg24  Based on the JavaDoc they do not thro Exception https://javadoc.io/doc/commons-io/commons-io/2.14.0/org/apache/commons/io/IOUtils.html.  It looks like they throw [UnsupportedCharsetException](https://docs.oracle.com/javase/8/docs/api/java/nio/charset/UnsupportedCharsetException.html)